src/app/auth/resetPassword/styles.tsx

```diff
@@ -0,0 +1,31 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+  container: {
+    paddingVertical: 63,
```

Comment:
Figma can be weird sometimes with pixel values, but usually pixel values will be multiples of 4, so changing this should be 64. If you see a pixel value that seems really close to a multiple of 4, its usually a good idea to round to that value.